src/averageAges.js

```diff
@@ -14,12 +14,20 @@
  *
  * @return {number}
  */
+function getAverageAgeValue(people) {
+  const age = people.map((person) => person.died - person.born);
```

Comment:
it's not age, it's array of ages, let's create more understandable variable name